internal/bots/slack/services.go

```diff
@@ -0,0 +1,33 @@
+package slack
+
+import (
+	"context"
+
+	"github.com/nekomeowww/insights-bot/ent/slackoauthcredentials"
+)
+
+func (b *SlackBot) createNewSlackCredential(teamID, accessToken string) error {
+	affectRows, err := b.ent.SlackOAuthCredentials.Update().
+		Where(slackoauthcredentials.TeamID(teamID)).
+		SetAccessToken(accessToken).
+		Save(context.Background())
+
+	if err != nil {
+		b.logger.WithError(err).Warn("slack: failed to update access token")
+		return err
+	}
```

Comment:
```suggestion
	affectRows, err := b.ent.SlackOAuthCredentials.Update().
		Where(slackoauthcredentials.TeamID(teamID)).
		SetAccessToken(accessToken).
		Save(context.Background())
	if err != nil {
		b.logger.WithError(err).Warn("slack: failed to update access token")
		return err
	}
```

中间没空行是会报错嘛？是的话这个算 false postive